lib/ruby_lsp/setup_bundler.rb

```diff
@@ -13,107 +14,133 @@ module RubyLsp
   class SetupBundler
     extend T::Sig
 
+    class BundleNotLocked < StandardError; end
+
     sig { params(project_path: String).void }
     def initialize(project_path)
       @project_path = project_path
-      @dependencies = T.let(load_dependencies, T::Hash[String, T.untyped])
-      @custom_bundle_dependencies = T.let(
-        if File.exist?(".ruby-lsp/Gemfile.lock")
-          Bundler::LockfileParser.new(Bundler.read_file(".ruby-lsp/Gemfile.lock")).dependencies
-        else
-          {}
+
+      # Custom bundle paths
+      @c_dir = T.let(Pathname.new(".ruby-lsp").expand_path(Dir.pwd), Pathname)
+      @c_bundle = T.let(@c_dir + "Gemfile", Pathname)
```

Comment:
nit: `@bundle` and `@lock` (and their `c_*` versions) would probably be more readable as `@gemfile`, `@lockfile`, and `@custom_*`.

Reply:
Renamed the variables!